widgets/style_video.py

```diff
@@ -1,15 +1,22 @@
+import random
+import urllib
+import requests
 from PyQt6.QtCore import Qt, QUrl, QObject, pyqtSignal, QThread
 from PyQt6.QtGui import QPixmap, QIcon
 from PyQt6.QtMultimedia import QMediaPlayer
 from PyQt6.QtMultimediaWidgets import QVideoWidget
 from PyQt6.QtWidgets import QWidget, QHBoxLayout, QVBoxLayout, QLabel, QPushButton, QGridLayout, QSlider, QFileDialog, \
     QStyle, QProgressBar
 
+from logic.preprocessing import preprocess_image, load_img_from_url, load_img
 from logic.style_transfer import StyleTransfer
 
 
 class StyleVideoMenu(QWidget):
     """Class defining GUI for video style transfer module"""
+
+    is_style_image_from_url = False
```

Comment:
jest jakiś powód że zmienna jest statyczna?

Reply:
jedyny powód to taki że w StyleImageMenu jest tak samo ale w ogólności to faktycznie jest to raczej niepotrzebne